pkg/internal/ebpf/common/spanner.go

```diff
@@ -95,3 +101,45 @@ func extractTraceparent(traceparent [55]byte) string {
 	}
 	return string(traceparent[:])
 }
+
+func getSQLOperationAndTable(queryString string) (string, string) {
+	fields := strings.Fields(queryString)
+	if len(fields) == 0 {
+		return "", ""
+	}
+	operation := strings.ToUpper(fields[0])
+	if len(fields) < 2 {
+		return operation, ""
+	}
+	table := ""
+	if operation == "UPDATE" {
+		return operation, fields[1]
+	}
+	if len(fields) < 3 {
+		return operation, ""
+	}
+	switch operation {
+	case "ALTER", "CREATE", "DROP", "TRUNCATE":
+		if strings.ToUpper(fields[1]) == "TABLE" {
```

Comment:
I think we can fix the cycops by extracting in another function, if this code is needed it shouldn't be commented out.

Reply:
This has been refactored to use new sqlprune package.